lib/printfrr.h

```diff
@@ -292,6 +294,8 @@ struct va_format {
 #pragma FRR printfrr_ext "%pTS" (struct timespec *)
 #pragma FRR printfrr_ext "%pTV" (struct timeval *)
 #pragma FRR printfrr_ext "%pTT" (time_t *)
+
+#pragma FRR printfrr_ext "%pIS" (uint8_t *id)
```

Comment:
The expression in braces here is a type, not a variable/parameter — you need to remove the `id`. (That's where the `unexpected identifier` warning comes from.)

Reply:
done